src/config/telemetry-config.json

```diff
@@ -17,6 +21,15 @@
     "redhat.java": {
         "enabled": "all",
         "ratio": "0.5",
+        "includes": [
+            {
+                "name" : "startup",
+                "dailyLimit": 1000
+            }, 
+            {
+                "name" : "*"
+            }
```

Comment:
required to include other events. This is a design error we'll need to address, but this config change is required for backward compatibility